### ISSUE TITLE
Cannot upgrade to the new 1.5.1 on Mac

### ISSUE THREAD

**Issue by drlaporte** [gp-rank13/gp-radiobuttons#2]
When I try to use the new 1.5.1 package, after putting the new .dylib file in the Extensions folder, MacOS tells me that the developer cannot be identified, therefore it does not let Gig Performer load the extension.

I don't remember to have encountered that when using the 1.5 version.

Thanks for your help - and for building that extension in the first place. It was working well until I installed GP 4.7 this morning. And the new radio button they propose is not yet as powerful as yours (e.g. handling variations).

**Comment by gp-rank13** [comment]
MacOS has added a few hurdles. You will need to go into System Preferences > Privacy & Security. Scroll down until you see a reference to the extension and choose to 'Allow Anyway'. The next time you open Gig Performer, you will get a different message, confirming if you would like to Open it.

**Comment by drlaporte** [comment]
Hi gp-rank13, I succeeded to open it taking a slightly different path as I was not seeing the extension listed in the Preferences. I followed the idea I found in a post to force Open an app and applied it to Open the .dylib file. I got a prompt warning me about the extension and asking me if I wanted to open it anyway, which I agreed upon. The Open led to opening Terminal that I quit. Then I launched GP which asked me if I wanted to load the extension and I said yes.
